config.ts

```diff
@@ -13,21 +13,25 @@ export const config = {
     showDebug: true,
     l1: {
       chain: goerli,
+      chainLogo: '/eth.svg',
```

Comment:
(arbitrary place for a comment)

I am also seeing the logo overextend the container.

![Screenshot from 2023-11-08 10-49-13](https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/12705146/fe5bca76-c78f-47e7-8cdb-6c4ae3cb640b)

Reply:
this is certainly an issue. I wonder if it's better dealth with in a different PR that sorta restyles each box and maybe reorients them on top of one another. i have a light preference to keep this PR limited to just the styling of L2 tokens.

Reply:
Okay, making an issue sounds good to me

Reply:
Created an issue here: https://github.com/ScopeLift/l2-flexible-voting-frontend/issues/112